hunspell: tolerate REP rule count mismatches

rmuir opened the pull request:
Similar to support for tolerating PFX/SFX count mismatches, add the ability to tolerate REP count mismatches.

The issue arises in recent updates to LibreOffice mongolian dictionary and is currently failing all PRs that change the analyzers:

https://bugs.documentfoundation.org/show_bug.cgi?id=164366

rmuir committed:
hunspell: tolerate REP rule count mismatches

Similar to support for tolerating PFX/SFX count mismatches, add the
ability to tolerate REP count mismatches.

The issue arises in recent updates to LibreOffice mongolian dictionary
and is currently failing all PRs that change the analyzers:

https://bugs.documentfoundation.org/show_bug.cgi?id=164366

donnerpeter reviewed (approved):
LGTM overall, thank you! Update CHANGES.txt?

rmuir committed:
CHANGES

rmuir commented:
I added a CHANGES. Also the underlying issue is now fixed in the LibreOffice. I will try to chase it down in the bataak repo after brushing up on some mongolian :)
